### ISSUE TITLE
Bug: Local Vrm upload cancel not resetting page state

### ISSUE THREAD

**Issue by danyyil-pelekhatyi** [semperai/amica#94]
Path to reproduce:
1) Go to website
2) Upload VRM using Load Vrm feature
3) Click Share
4) On share page click Upload VRM button
5) Before upload starts click round downloading icon to cancel upload

Actual result:
Empty Vrm Url field and empty Vrm Demo shown

Expected result:
Page resets to previous state before download: Vrm Url field and Vrm Demo are hidden, Upload Local VRM message and button is shown 

Basically, we need to call this code on cancel:
```typescript
if (vrmLoadingFromIndexedDb)
    setVrmLoadingFromIndexedDb(false);
```
